python_files/nivo_outcomes.py

```diff
@@ -18,83 +17,15 @@
-#
-# # look at evolution
-# evolution_df = pd.read_csv(os.path.join(data_dir, 'nivo_outcomes/evolution_df.csv'))
-# evolution_df = evolution_df.merge(patient_metadata[['Patient_ID', 'Induction_treatment', 'Time_to_progression_weeks_RECIST1.1', 'Censoring_PFS_RECIST1.1', 'Clinical_benefit']].drop_duplicates(), on='Patient_ID', how='left')
-# evolution_df = evolution_df.rename(columns={'raw_value': 'raw_mean', 'normalized_value': 'normalized_mean', 'comparison': 'Timepoint'})
-# evolution_df = evolution_df[['feature_name_unique', 'raw_mean', 'normalized_mean', 'Patient_ID', 'Timepoint', 'Induction_treatment', 'Time_to_progression_weeks_RECIST1.1', 'Censoring_PFS_RECIST1.1', 'Clinical_benefit']]
-#
-# # combine together into single df
-# combined_df = timepoint_features.copy()
-# combined_df = combined_df[['feature_name_unique', 'raw_mean', 'normalized_mean', 'Patient_ID', 'Timepoint', 'Induction_treatment', 'Time_to_progression_weeks_RECIST1.1', 'Censoring_PFS_RECIST1.1', 'Clinical_benefit']]
-# combined_df = pd.concat([combined_df, evolution_df[['feature_name_unique', 'raw_mean', 'normalized_mean',
-#                                                     'Patient_ID', 'Timepoint', 'Induction_treatment', 'Time_to_progression_weeks_RECIST1.1', 'Censoring_PFS_RECIST1.1', 'Clinical_benefit']]])
-# combined_df['combined_name'] = combined_df.feature_name_unique + '__' + combined_df.Timepoint
-#
-# combined_df.to_csv(os.path.join(data_dir, 'nivo_outcomes/combined_df.csv'), index=False)
+base_dir = '/Volumes/Shared/Noah Greenwald/TONIC_Cohort/'
 
-# load previously computed results
-combined_df = pd.read_csv(os.path.join(data_dir, 'nivo_outcomes/combined_df.csv'))
 
+harmonized_metadata = pd.read_csv(os.path.join(base_dir, 'intermediate_files/metadata/harmonized_metadata.csv'))
+patient_metadata = pd.read_csv(os.path.join(base_dir, 'intermediate_files/metadata/TONIC_data_per_patient.csv'))
+feature_metadata = pd.read_csv(os.path.join(base_dir, 'analysis_files/feature_metadata.csv'))
 
-# # look at change due to nivo
-# for comparison in ['baseline__on_nivo', 'baseline__post_induction', 'post_induction__on_nivo']:
-#
-#     # compare pre and post therapy
-#     pop_1, pop_2 = comparison.split('__')
-#     compare_df = compare_timepoints(feature_df=timepoint_features, timepoint_1_name=pop_1, timepoint_1_list=[pop_1],
-#                                     timepoint_2_name=pop_2, timepoint_2_list=[pop_2], paired=comparison)
-#     # plot results
-#     output_dir = plot_dir + '/evolution_{}'.format(comparison)
-#     if not os.path.exists(output_dir):
-#         os.makedirs(output_dir)
-#
-#     summarize_timepoint_enrichment(input_df=compare_df, feature_df=timepoint_features, timepoints=[pop_1, pop_2],
-#                                  pval_thresh=2, diff_thresh=0.3, output_dir=output_dir)
+# load previously computed results
+combined_df = pd.read_csv(os.path.join(base_dir, 'analysis_files/timepoint_combined_features.csv'))
```

Comment:
I renamed `combined_df.csv` to `timepoint_combined_features.csv` so the filename is more informative. I think I made all the substitutions, but not sure.